force-app/main/default/lwc/donationHistoryFilter/donationHistoryFilter.js

```diff
@@ -0,0 +1,63 @@
+import { api, LightningElement, track } from 'lwc';
+import donationHistoryLabelLifetime from '@salesforce/label/c.donationHistoryLabelLifetime'
+import donationHistoryFilterSelectAYear from '@salesforce/label/c.donationHistoryFilterSelectAYear'
+import getYearsWithDonation from '@salesforce/apex/DonationHistoryController.getYearsWithDonations'
+
+const LIFETIME_VALUE = 'lifetime';
+export default class DonationHistoryFilter extends LightningElement {
+    
+    labels = {
+        lifetime : donationHistoryLabelLifetime,
+        donationHistoryFilterSelectAYear : donationHistoryFilterSelectAYear
+    }
+
+    value = LIFETIME_VALUE;
+    
+    @track
+    options = [];
+
+    @api
+    contactId
```

Comment:
Can this be moved up to the top of the component definition too?
```suggestion
    @api contactId;
    
```